### PR TITLE
Add 'Faith Above Reason' topic to recomment. Remove Group Articles an…

### PR DESCRIPTION
**Pull request by bbfsdev**
…d Shamati. Replace popular suggestor with random suggestor.

### CODE REVIEW

File: src/sagas/recommended.js [hunk 1]
```diff
@@ -152,11 +152,12 @@ export function* fetchRecommended(action) {
     // Append predefined, constant recommentations.
     if (variant === AB_RECOMMEND_NEW) {
       data.feeds.splice(data.feeds.length - 1, 0, [
-        { content_type: CT_SOURCE, uid: 'itcVAcFn' },         // Maamar Ha-Arvut
+        /*{ content_type: CT_SOURCE, uid: 'itcVAcFn' },/*     // Maamar Ha-Arvut
```
Comment (davgur) [inline src/sagas/recommended.js:155]: are you make it special (use /* instead */)?

File: src/components/Pages/Unit/widgets/Recommended/Main/DisplayRecommended.js [hunk 2]
```diff
@@ -123,19 +123,27 @@ const RecommendedPlaylist = (recommendForUnit, units, selected, t, chronicles, v
                     asList
                     label={unitLabels[index]}
                     size={'small'} />
-                  : (IsUnitContentType(unit.content_type) ?
+                  : (unit.content_type && IsUnitContentType(unit.content_type) ?
```
Comment (davgur) [inline src/components/Pages/Unit/widgets/Recommended/Main/DisplayRecommended.js:126]: you don't want add this check on IsUnitContentType?

**Review by davgur — APPROVED** [review]

File: src/sagas/recommended.js (hunk 1 above)
Comment (bbfsdev) [reply src/sagas/recommended.js:155]: Fixed.

File: src/components/Pages/Unit/widgets/Recommended/Main/DisplayRecommended.js (hunk 2 above)
Comment (bbfsdev) [reply src/components/Pages/Unit/widgets/Recommended/Main/DisplayRecommended.js:126]: No, as here we check simple string which the the content_type string. unit may be different things ,e.g., lesson, topic, source ....

**Pull request merged**